lib/ui/collections_gallery_widget.dart

```diff
@@ -24,6 +24,8 @@ import 'package:photos/utils/local_settings.dart';
 import 'package:photos/utils/navigation_util.dart';
 import 'package:photos/utils/toast_util.dart';
 
+import 'archive_page.dart';
```

Comment:
`package:photos/...` so that even if we extract code to external libs, imports will not have to change.